utils.js

```diff
@@ -53,6 +59,24 @@ function post(url, jar, form) {
   return request(op).then(function(res) {return res[0];});
 }
 
+function postFormData(url, jar, form, qs) {
```

Comment:
Mmmh I'm not sure this is useful as a different function from `post`. Can we maybe add a 4th argument to `post`? @Schmavery thoughts?